static/css/devhub/new-landing/base.less

```diff
@@ -3,32 +3,28 @@
 
 // Grid
 @grid-max: 960px;
-@extra-small: ~"only screen and (max-width: 399px)";
-@small: ~"only screen and (min-width: 400px)";
-@medium: ~"only screen and (min-width: 700px)";
-@large: ~"only screen and (min-width: 1024px)";
+@large: ~"only screen and (min-width: 960px)";
```

Comment:
The change from the 100% width mobile layout seems to happen too late resulting in a lot of white-space for this view (see the screenshot below) you could switch to larger viewport styling at a lower width value. 

<img width="967" alt="Developer_Hub____Add-ons_for_Firefox" src="https://user-images.githubusercontent.com/1514/66417288-8f301e00-e9f7-11e9-8de2-691da85ab72e.png">

Reply:
> The change from the 100% width mobile layout seems to happen too late resulting in a lot of white-space for this view (see the screenshot below) you could switch to larger viewport styling at a lower width value.  

What width?  Should I reinstate `@medium` at 700px wide?